vassal-app/src/main/java/VASSAL/counters/Footprint.java

```diff
@@ -270,6 +273,7 @@ public Object getProperty(Object key) {
    */
   @Override
   public void setMoved(boolean justMoved) {
+    requireNoOuterRotate();
```

Comment:
This only needs to be checked once per game.

Reply:
Any suggestions how to track that? Just some boolean field that is constructored to false and we set to true when we've done the check?

Reply:
Just a private boolean rotateCheckedOnMove = false and set it true when you check. I don't see any reason why it should need to be reset. Except possibly if you refresh the counters, but I wouldn't worry about that, it will be picked up the next time.

Reply:
okay, fixed up.